packages/jazz-react/src/auth/DemoAuth.tsx

```diff
@@ -81,16 +81,19 @@ export const DemoAuthBasicUI = ({
       ? window.matchMedia("(prefers-color-scheme: dark)").matches
       : false;
 
-  if (user && state.state === "ready") {
+  const isAutoLogin = !!(user && state.state === "ready");
```

Comment:
@gdorsi The `Boolean()` gave me TS errors that don't make sense to me
<img width="700" alt="image" src="https://github.com/user-attachments/assets/9bafdc85-299d-4e15-af2e-2cc84df22d9b">